packages/arb-avm-go/vm/instructions.go

```diff
@@ -122,37 +123,38 @@ func (w BlockedError) Error() string {
 	return "VMBlockederror"
 }
 
-func RunInstruction(m *Machine, op value.Operation) (StackMods, machine.BlockReason) {
+func RunInstruction(m *Machine, op value.Operation) (StackMods, uint64, machine.BlockReason) {
```

Comment:
It doesn't look like the gas return value is used anywhere. It might be better to remove this.